cluster/gce/gci/configure-helper.sh

```diff
@@ -2104,6 +2104,11 @@ EOF
     prepare-kube-proxy-manifest-variables "$src_dir/kube-proxy/kube-proxy-ds.yaml"
     setup-addon-manifests "addons" "kube-proxy"
   fi
+  # Setup prometheus stack for monitoring kubernetes cluster
+  if [[ "${ENABLE_PROMETHEUS_MONITORING:-}" == "true" ]]; then
```

Comment:
Have you tested it? If you add new env var, I think you need to add it here as well: https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/util.sh#L678

Reply:
Just finished testing it from fresh cluster. Moving prometheus out of cluster-monitoring took more changes.